.github/tekton/pull-request.yaml

```diff
@@ -0,0 +1,148 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: ks-pull-request-pipeline
+spec:
+  workspaces:
+    - name: repo
+  params:
+    - name: repo-full-name
+      description: "Repository full name. like: kubesphere-sigs/ks"
+    - name: clone-url
+      description: Git repository clone URL.
+    - name: revision
+      description: Git repository revision to checkout.
+    - name: dashboard-url
+      description: Tekton dashboard access URL.
+      default: "http://demo:31962/#/namespaces/ks/pipelineruns"
```

Comment:
Please use a correct IP address.

Reply:
This is just a demo value, I will move it to description instead of being default value.